docs/workload/elasticsearch/elasticsearch.md

```diff
@@ -0,0 +1,573 @@
+- Choose **OpenEBS Local PV**, if you are not requiring replicated storage but high performance storage engine. The steps for deploying Prometheus operator using OpenEBS Local PV can be found [here](https://docs.openebs.io/docs/next/elasticsearch.html).
+
+
+### Configure OpenEBS cStor StorageClass
+
+The Storage Class `cstorsc-gqzxe`, which was created during cStor configuration, has been chosen to deploy Elasticsearch in the Kubernetes cluster. Please use the cStor storage class name which you had created during cStor configuration.
+
+
+###  Installing KUDO Operator
+
+In this section, we will install the KUDO operator. We will later deploy the latest available version of Elasticsearch using KUDO. 
+
+Use the latest stable version of KUDO CLI. The latest version of KUDO can be checked from [here](https://github.com/kudobuilder/kudo/releases).
+
+#### Verify if Cert-manager is installed
+
+For installing KUDO operator, the Cert-manager must be already installed in your cluster. If not, install the Cert-manager. The instruction can be found from [here](https://cert-manager.io/docs/installation/kubernetes/#installing-with-regular-manifests). Since our K8s version is v1.18.12, we have installed Cert-manager using the following command.
+
+```bash
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.2.0/cert-manager.yaml
+```
+
+```bash
+kubectl get pods --namespace cert-manager
+```
+```bash
+#sample output
+NAME                                      READY   STATUS    RESTARTS   AGE
+cert-manager-7747db9d88-7qjnm             1/1     Running   0          81m
+cert-manager-cainjector-87c85c6ff-whnzr   1/1     Running   0          81m
+cert-manager-webhook-64dc9fff44-qww8s     1/1     Running   0          81m
+```
+
+#### Installing KUDO operator into cluster
+
+Once prerequisites are installed you need to initialize the KUDO operator. The following command will install KUDO v0.18.2.
+
+```bash
+kubectl-kudo init --version 0.18.2
+```
+```bash
+#sample output
+$KUDO_HOME has been configured at /home/k8s/.kudo
+✅ installed crds
+✅ installed namespace
+✅ installed service account
+✅ installed webhook
+✅ installed kudo controller
+```
+
+Verify pods in the `kudo-system` namespace:
+```bash
+kubectl get pod -n kudo-system
+```
+```bash
+#sample output
+NAME                        READY   STATUS    RESTARTS   AGE
+kudo-controller-manager-0   1/1     Running   0          25s
+```
+
+#### Setting cStor Storage Class as default
+
+Change the default storage class from your current setting to OpenEBS cStor. For example, in this tutorial default storage class is used as `cstorsc-gqzxe` from standard.
```

Comment:
You may need to mention like below?
For example, in this tutorial default storage class is used as `cstorsc-gqzxe` from the default storage class in GKE `standard`.

Reply:
Also, it will be good to mention why we need to make the cStor storage class as default. 
My understanding is as follows.
If we mention default storage class is cStor, then we don't need to mention in the Elasticsearch command for its components and it will take it automatically for all of its components. Is it correct?

Reply:
Changed the description to match kudo operator